pkg/Makefile

```diff
@@ -41,6 +41,13 @@ osc-cli-x86_64.zip: osc-cli
 	zip -r osc-cli-x86_64.zip osc-cli
 
 osc-cli:
```

Comment:
rename to osc-cli-windows, no ?

Reply:
well, I could, but as it was here before, I guess it should be done in another ticket PR.